src/map/MapSidebar.js

```diff
@@ -97,6 +126,138 @@ function MapSidebarMain (props) {
+    async function getNextMeetupDate(){
+      const meetupLocations = await api.rpc.encointer.getLocations(cid);
+      const tempLocation = locationFromJson(api, meetupLocations[0]);
+      const temporaryTime = await getNextMeetupTime(api, tempLocation);
+      const tempdate = new Date(temporaryTime.toNumber());
+      const temparray = (tempdate.toString()).split("G");
+      const tempresultarray = temparray[0].split(" ");
+      setNextMeetupTime (" "+tempresultarray[0]+" "+tempresultarray[2]+" "+tempresultarray[1]+ " "+tempresultarray[3]);
+    }
+    getNextMeetupDate();
+  }, [api,nextMeetupTime, setNextMeetupTime, cid]);
+
+  // gets the nominal Income of a Community
+  const [nominalIncome, setNominalIncome]= useState([]);
+  useEffect(() => {
+    async function getNominalIncome(){
+      const tempNominalIncome = await getCeremonyIncome(api, cid);
+      return parseI64F64(tempNominalIncome);
+    }
+    let isMountedNominal = true 
+    getNominalIncome().then((data) => {
+      if (isMountedNominal) setNominalIncome(" "+data);
+    });
+    return () => {isMountedNominal = false; };
+
+  },[nominalIncome, setNominalIncome,getCeremonyIncome, api, cid])
+
+  const [registeredBootstrappers, setregisteredBootstrappers]= useState([]);
+  const [registeredReputables, setregisteredReputables]= useState([]);
+  const [registeredEndorseees, setregisteredEndorsees]= useState([]);
+  const [registeredNewbies, setregisteredNewbies]= useState([]);
+  const [unassignedNewbies, setunassignedNewbies]= useState([]);
+  
+// gets the data of all people that registered for a Ceremony
+  function showPhasepeopleData(){
+    useEffect(() => {
+      async function getPhasepeopleData(){
+        const currentCeremonyIndex = await api.query.encointerScheduler.currentCeremonyIndex();
+        const currentCommunityCeremony = new CommunityCeremony(api.registry, [cid, currentCeremonyIndex])
+        const [tempregisteredBootstrappers, tempregisteredReputables, tempregisteredEndorseees,tempregisteredNewbies, assignmentCounts] = await Promise.all([
+          api.query.encointerCeremonies.bootstrapperCount(currentCommunityCeremony),
+          api.query.encointerCeremonies.reputableCount(currentCommunityCeremony),
+          api.query.encointerCeremonies.endorseeCount(currentCommunityCeremony),
+          api.query.encointerCeremonies.newbieCount(currentCommunityCeremony),
+          api.query.encointerCeremonies.assignmentCounts(currentCommunityCeremony)
+        ]);
+        return [tempregisteredBootstrappers, tempregisteredReputables, tempregisteredEndorseees, tempregisteredNewbies, assignmentCounts]
+      }
+
+      let isMounted = true;
+      getPhasepeopleData().then((data) => {
+        if(isMounted){
+        setregisteredBootstrappers(data[0]);
+        setregisteredReputables(data[1]);
+        setregisteredEndorsees(data[2]);
+        setregisteredNewbies(data[3]);
+        setunassignedNewbies(registeredNewbies-data[4].newbies);
```

Comment:
Hmm, it seems that there is an error. I see -9 in the current state:

![image](https://user-images.githubusercontent.com/37865735/193057118-71e07da9-2d85-4486-9ac7-5c665a8b295b.png)


It seems that `registeredNewbies` is 0 at this point.

Reply:
need to look, how it looks, when it's in the assigning and attesting phase. might have found a fix, but doubt it.